packages/react/src/themes/teams-high-contrast/components/Menu/menuItemStyles.ts

```diff
@@ -22,10 +22,10 @@ const menuItemStyles: ComponentSlotStylesInput<MenuItemPropsAndState, MenuVariab
       }),
 
       ...((iconOnly || vertical) && {
-        ...(isFromKeyboard && {
+        '[data-focus-visible-added]': {
```

Comment:
Per your suggestion about the plugin, I think we should stick with the standard `:focus-visible` pseudo selector.  

The style files should be as vanilla as possible.  When compiled to CSS sheets, users can either use our plugin or use their own CSS processor to handle their conversions.  Eventually, the `:focus-visible` will just work without any plugins or processors since it is in the CSS spec.